examples/input/gamepad_input_events.rs

```diff
@@ -17,31 +18,46 @@ fn main() {
 
 fn gamepad_events(
     mut connection_events: EventReader<GamepadConnectionEvent>,
+    // Handles the continuous measure of an axis, equivalent to GamepadAxes::get.
+    // Whenever that *raw* value changes, this event is emitted.
+    mut raw_axis_changed_events: EventReader<RawGamepadAxisChangedEvent>,
```

Comment:
I think covering the raw events here adds too much conceptual noise to this example. I'm ok with leaving raw events as an advanced / slightly more obscure feature.